packages/main/pyproject.toml

```diff
@@ -59,7 +59,7 @@ exchangelib = "^3.1.1"
 xlrd = "^2.0.1"
 xlwt = "^1.3.0"
 xlutils = "^2.0.0"
-openpyxl = "^3.0.3"
+openpyxl = "^3.0.9"
```

Comment:
I don't think we actually need to _depend_ on a newer version, but bumping the patch version doesn't hurt. The problem was the `poetry lock` generated lockfile that had a deleted version, and it could've been re-locked to fix.

Just being a bit pedantic here since these version ranges affect the wheels and the lockfiles don't. Probably no need to change it back though.